docs_src/examples/Ch-ExamplesProvisionDevice.md

```diff
@@ -14,36 +14,23 @@ documents for API framing (RAML docs for the EdgeX services may be found
 in src/test/resources/raml or on the wiki). Note that this specific
 example can be tweaked for use with the other Device Services.
 
-1.  Upload the device profile above to metadata with a POST to
```

Comment:
why do you eliminate the profile upload step?

Reply:
Added it back.